binding/dotnet/Porcupine/Porcupine.csproj

```diff
@@ -58,10 +58,17 @@
             <Link>lib\windows\amd64\libpv_porcupine.dll</Link>
             <Visible>false</Visible>
         </Content>
+        <Content Include="..\..\..\lib\windows\arm64\libpv_porcupine.dll">
```

Comment:
just an organizational thing, but make sure all the platforms are under their correct heading - e.g. windows arm should not be under the .net standard 2.0 heading, but should be under the .net 6 heading